actors/builtin/miner/miner_state.go

```diff
@@ -32,7 +33,7 @@ type State struct {
 	PreCommitDeposits abi.TokenAmount // Total funds locked as PreCommitDeposits
 	LockedFunds       abi.TokenAmount // Total rewards and added funds locked in vesting table
 
-	VestingFunds cid.Cid // Array, AMT[ChainEpoch]TokenAmount
+	VestingFunds cid.Cid // Vesting Funds schedule for the miner. Slice[(VestingEpoch, VestingAmount)]
```

Comment:
It's not actually a Slice, it's a `VestingFunds`. I see how, given the name, this could be inferred, but please specify for consistency with other CIDs.

Reply:
This is done.

Reply:
This is done.